bench/utils.py

```diff
@@ -300,14 +300,19 @@ def setup_app(app):
 		setup_app(app)
 
 
-def exec_cmd(cmd, cwd='.'):
+def exec_cmd(cmd, cwd='.', fail_on_non_zero_return=False):
```

Comment:
`fail_on_non_zero_return` is wordy, yet fails to explain what actually would happen. If an exception would be raised, maybe rename this to `throw` or `raise`?